apps/ui/src/networks/offchain/api/types.ts

```diff
@@ -33,10 +35,18 @@ export type ApiProposal = {
   space: {
     id: string;
     name: string;
+    network: string;
     admins: string[];
     symbol: string;
   };
-  type: 'basic' | 'single-choice' | string;
+  type:
+    | 'basic'
+    | 'single-choice'
+    | 'approval'
+    | 'ranked-choice'
+    | 'quadratic'
+    | 'weighted'
+    | 'custom';
```

Comment:
What is 'custom' here?

Reply:
From here https://github.com/snapshot-labs/snapshot.js/blob/f77d09ab8ddaede558dd8fe693d194598a7da7a3/src/schemas/proposal.json#L44